consensus/fuzz/fuzz_targets/mocks/fuzzer.rs

```diff
@@ -0,0 +1,407 @@
+//! Byzantine participant that sends random messages.
+
+use arbitrary::{Arbitrary, Unstructured};
+use commonware_codec::{Decode, Encode};
+use commonware_consensus::{
+    simplex::{
+        mocks::supervisor::Supervisor,
+        types::{Finalize, Notarize, Nullify, Proposal, Voter},
+    },
+    Supervisor as SupervisorTrait, Viewable,
+};
+use commonware_cryptography::{
+    ed25519::{PrivateKey, PublicKey},
+    sha256::Digest as Sha256Digest,
+    Digest, Signer as _,
+};
+use commonware_macros::select;
+use commonware_p2p::{Receiver, Recipients, Sender};
+use commonware_runtime::{Clock, Handle, Spawner};
+use futures_timer::Delay;
+use rand::{rngs::StdRng, RngCore, SeedableRng};
+use std::time::Duration;
+
+const DEFAULT_TIMEOUT: Duration = Duration::from_millis(500);
+
+#[derive(Debug, Clone, Arbitrary)]
+pub enum PartitionStrategy {
```

Comment:
It would be great to move this with the `commonware-p2p::simulated` helpers (as called out before)